.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,132 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+  schedule:
+    - cron: "0 17 * * *"
```

Comment:
<img width="679" alt="Screen Shot 2020-01-29 at 22 57 59" src="https://user-images.githubusercontent.com/605483/73401240-0eb96400-42eb-11ea-860c-150e9d4f0811.png">

Looks like there's a cronjob scheduled to run once a day, doesn't it?

Reply:
Currently, the master branch is run every day. I'm fine running the master branch daily, but would prefer if we can also add the latest stable branch (7.0.x at the moment) every week.

Reply:
I believe that this is currently not possible, see https://help.github.com/en/actions/automating-your-workflow-with-github-actions/events-that-trigger-workflows#scheduled-events-schedule:

> You can schedule a workflow to run at specific UTC times using POSIX cron syntax. **Scheduled workflows run on the latest commit on the default or base branch**. The shortest interval you can run scheduled workflows is once every 5 minutes.

Reply:
Ah, I see. That's fine for now :)